Makd.mak

```diff
@@ -652,9 +652,9 @@ d2conv: $O/d2conv.stamp
 $O/d2conv.stamp: $C
 	$Vfind $C -type f -regex '^.+\.d$$' > $@
 ifeq "$(shell d1to2fix --help | grep -- --input)" ""
```

Comment:
If you want to keep the current structure, the easiest fix is: `$(shell d12tofix --help 2>/dev/null | grep -- --input)`. I think is a bit ugly to try to run `d1to2fix` just by parsing the Makefile, but I won't insist a lot on the which solution to use. Is a transitional step anyway...